packages/instant/src/components/ui/text.tsx

```diff
@@ -23,14 +23,11 @@ export interface TextProps {
     display?: string;
 }
 
-const PlainText: React.StatelessComponent<TextProps> = ({ children, className, onClick }) => (
-    <div className={className} onClick={onClick}>
-        {children}
-    </div>
-);
-
 const darkenOnHoverAmount = 0.3;
-export const Text = styled(PlainText)`
+export const Text =
```

Comment:
thanks for changing these!  feels cleaner, but bummer that prettier makes it look funky